seismic-app/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# base image
+FROM node:12.2.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install and cache app dependencies
+COPY package.json /app/package.json
```

Comment:
@tudoramariei You need to copy both `package.json` and `package-lock.json`, to make sure everyone is working on the same versions and we don't have any surprises (not everyone is using SemVer for their packages).

Reply:
(not so) fun enough, if I add `COPY package.json /app/package-lock.json`, it seems to crash with the following error:
```log
ERROR: Service 'seismic-app' failed to build: The command '/bin/sh -c npm install' returned a non-zero code: 1
```

Reply:
@tudoramariei that's to be expected since you are basically renaming `package.json` to `package-lock.json` inside the container.
Try this: ```COPY package*.json ./```

Reply:
OK, I saw the typo now 😅